src/hotspot/cpu/x86/templateTable_x86.cpp

```diff
@@ -4184,12 +4181,12 @@ void TemplateTable::instanceof() {
   __ pop_ptr(rdx); // restore receiver
   __ verify_oop(rdx);
   Register tmp_load_klass = LP64_ONLY(rscratch1) NOT_LP64(noreg);
-  __ load_klass(rdx, rdx, tmp_load_klass);
-  __ jmpb(resolved);
+  __ load_klass(rdx, rdx, tmp_load_klass, false);
+  __ jmp(resolved);
```

Comment:
Have you run into troubles with `jmpb` short branch here?

Reply:
Yes. The (subsequent) load_klass() increases the distance to resolved.

Reply:
OK